src/app/tests/suites/certification/Test_TC_ECAL_1_1.yaml

```diff
@@ -0,0 +1,180 @@
+
+    - label:
+          "Step 3d: Given ECAL.S.F02(AUXLD) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: ECAL.S.F02
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x3]
+
+    - label:
+          "Step 3e: Given ECAL.S.F03(PEAKP) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: ECAL.S.F03
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
+
+    - label: "Step 4a: Read the global attribute: AttributeList"
+      PICS: PICS_EVENT_LIST_ENABLED
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                         0x0000,
+                         0x0001,
+                         0x0002,
+                         0x0003,
+                         0x0004,
+                         0x0005,
+                         0x0006,
+                         0x0007,
+                         0x0008,
+                         0x0009,
+                         0xFFF8, # GeneratedCommandList
+                         0xFFF9, # AcceptedCommandList
+                         0xFFFA, # EventList
+                         0xFFFB, # AttributeList
+                         0xFFFC, # FeatureMap
+                         0xFFFD  # ClusterRevision                          
+                        ]
+
+    - label: "Step 4a: Read the global attribute: AttributeList"
+      PICS: "!PICS_EVENT_LIST_ENABLED"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                         0x0000,
+                         0x0001,
+                         0x0002,
+                         0x0003,
+                         0x0004,
+                         0x0005,
+                         0x0006,
+                         0x0007,
+                         0x0008,
+                         0x0009,
+                         0xFFF8, # GeneratedCommandList
+                         0xFFF9, # AcceptedCommandList
+                         0xFFFB, # AttributeList
+                         0xFFFC, # FeatureMap
+                         0xFFFD  # ClusterRevision                          
+                        ]
+
+    - label:
+          "Step 4b: Read the feature dependent(ECAL.S.F02) attribute in
+          AttributeList"
+      PICS: ECAL.S.F02
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                         0x000B, # PeakPeriodStatus
+                         0x000C, # PeakPeriodStartTime
+                         0x000D  # PeakPeriodEndTime
+                        ]
```

Comment:
These should be [0x000a # CurrentPeakPeriod, 0x000b # NextPeakPeriod], according to the PR#7829

Reply:
fixed